### PR TITLE
Improve Vulcanexus overview and developer tools documentation

### PR DESCRIPTION
**Pull request by irenebm**
## Description
This is a WIP.

### CODE REVIEW

File: docs/rst/introduction/introduction.rst [hunk 1]
```diff
@@ -37,22 +37,6 @@ Algorithms
 Another category includes robotic application's algorithms.
 *Vulcanexus*, being a *ROS 2* all-in-one tool set, provides the algorithms included in *ROS 2* ecosystem.
 
-Developer tools
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:40]: I'd not remove them from here.

File: docs/rst/introduction/metapackages.rst [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.. _vulcanexus_binary_distribution:
+.. _vulcanexus_metapackages:
 
-Vulcanexus binary distributions
-===============================
+Vulcanexus Metapackages
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:3]:
```suggestion
Vulcanexus metapackages
```

File: docs/rst/introduction/metapackages.rst [hunk 3]
```diff
@@ -31,8 +31,8 @@ Vulcanexus Tools
 ----------------
 
 **Vulcanexus Tools** is focused on helping developers with the introspection and debugging of their system.
-Thus, the `ROS 2 Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_ and the `ROS 2 Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_ are included in order to monitor and diagnose the *ROS 2* communication network.
-Also, `ROS 2 Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_ is provided as a simple demo to help testing that the communication is successfully established in the system.
+Thus, the :ref:`ROS 2 Monitor <vulcanexus_monitor>` and the `ROS 2 Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_ are included in order to monitor and diagnose the *ROS 2* communication network.
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:34]: Why the statistics backend does not have its own page?

```suggestion
Thus, the :ref:`ROS 2 Monitor <vulcanexus_monitor>` and the :ref:`ROS 2 Statistics Backend <vulcanexus_backend>` are included in order to monitor and diagnose the *ROS 2* communication network.
```

File: docs/rst/introduction/metapackages.rst [hunk 4]
```diff
@@ -1,9 +1,9 @@
-.. _vulcanexus_binary_distribution:
+.. _vulcanexus_metapackages:
 
-Vulcanexus binary distributions
-===============================
+Vulcanexus Metapackages
+=======================
 
-*Vulcanexus* provides several binary distributions that install specific packages depending on the developer's needs.
+*Vulcanexus* provides several installations with specific packages depending on the developer's needs.
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:6]: I'd not change the binary distributions here as it is what it is. Maybe a clarification of what a binary distribution is can be added here.

File: docs/rst/introduction/tools/shapes_demo.rst [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.. _vulcanexus_shapes_demo:
+
+ROS 2 Shapes Demo
+=================
+
+TODO
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/shapes_demo.rst:6]:
```suggestion
.. note:

    This page is under maintenance and will be updated soon. 
```

File: docs/rst/introduction/tools/webots.rst [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.. _webots:
+
+Webots
+======
+
+TODO
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/webots.rst:6]: Same note as before

File: docs/rst/tutorials/core/ros2_tutorials.rst [hunk 7]
```diff
@@ -3,6 +3,8 @@
 Vulcanexus Core Tutorials
 ==========================
 
+ROS 2 :doc:`tutorials <../../../ros2_documentation/source/Tutorials>` are provided in the ROS2 Documentation. *Vulcanexus* expands those tutorials with new ones that show user features, tools provided with *Vulcanexus* and advanced tutorials that are not explained in the ROS 2 Documentation.
```
Comment (rsanchez15) [inline docs/rst/tutorials/core/ros2_tutorials.rst:6]: Please remind to break line after every sentence. 
```suggestion
A set of :doc:`ROS 2 tutorials <../../../ros2_documentation/source/Tutorials>` are provided in the ROS 2 Documentation.
*Vulcanexus* documentation adds further ROS 2 tutorials, along with middleware specific user features only available in Fast DDS.
```

File: docs/index.rst [hunk 8]
```diff
@@ -11,16 +11,43 @@ Vulcanexus Documentation
...
+
+Vulcanexus project
+^^^^^^^^^^^^^^^^^^
+
+*Vulcanexus* is composed of a collection of downloadable packages:
 
 1. **VULCANEXUS-CORE**: a set of software libraries that enables users to build the most comprehensive and straightforward robotics application.
-   It consists of eProsima *Fast DDS* and *ROS 2*.
+   It consists of eProsima *Fast DDS* as middleware in *ROS 2*.
 2. **VULCANEXUS-TOOLS**: a set of features and applications which allows users to test, improve and configure the performance of *Vulcanexus* in their systems.
```
Comment (rsanchez15) [inline docs/index.rst:51]: I'd change the packages to lowercase format. This applies to all packages in this list.

```suggestion
2. **Vulcanexus Tools**: a set of features and applications which allows users to test, improve and configure the performance of *Vulcanexus* in their systems.
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/rst/introduction/tools/monitor.rst [hunk 9]
```diff
@@ -0,0 +1,44 @@
+.. _vulcanexus_monitor:
+
+ROS 2 Monitor
+=============
+
+*ROS 2 Monitor* is a graphical desktop application shipped with *Vulcanexus Tools* aimed at monitoring DDS environments
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/monitor.rst:6]: Do not break lines every 120 characters, just with `.`

File: docs/rst/introduction/tools/monitor.rst [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls>
+        <source src="../../../_static/monitor_vulcanexus.mp4">
+        Your browser does not support the video tag.
+    </video>
+
+Visit `eProsima Fast DDS Monitor Documentation <https://fast-dds-monitor.readthedocs.io/en/latest/>`_ for more
+information on how to use this application. A hands-on example is also available in :ref:`this <tutorials_tools_fastdds_monitor>`
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/monitor.rst:42]: Break line and link in the `link` word
```suggestion
information on how to use this application. 
A hands-on example is also available in this :ref:`link <tutorials_tools_fastdds_monitor>`
```

File: docs/rst/introduction/tools/router.rst [hunk 11]
```diff
@@ -0,0 +1,44 @@
+.. _vulcanexus_router:
+
+ROS 2 Router
+============
+
+*ROS 2 Router* (a.k.a. *DDS Router*) is an end-user software application shipped with *Vulcanexus Cloud* that enables
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/router.rst:6]: Same as before. Do not break lines every 120 chars, just on `.`

File: docs/rst/introduction/tools/router.rst [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+     topic specified by the user.
+  6. **Dynamic topic discovery**: the user does not need to fully specify over which topics to communicate (i.e. provide
+     concrete topic names and types). The discovery of topics matching the allowlisting rules automatically triggers the
+     creation of all entities required for communication.
+  7. **Quality of Service preservation**: *ROS 2 Router* uses the QoS set in the user's *ROS 2* network and keeps the
+     reliability and durability of the communication for each topic. These QoS are also manually configurable.
+
+.. figure:: /rst/figures/intro/tools/router/ddsrouter_cloud.png
+
+Visit `eProsima DDS Router Documentation <https://eprosima-dds-router.readthedocs.io/en/latest>`_ for more information
```
Comment (rsanchez15) [inline docs/rst/introduction/tools/router.rst:42]: I'd separate this into another section called "Next steps" or something like that. This also applies to the ROS 2 Monitor page. Create an overview section and a next steps section

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/rst/introduction/metapackages.rst (hunk 3 above)
Comment (juanlofer-eprosima) [reply docs/rst/introduction/metapackages.rst:34]: Using `Fast DDS Statistics Backend` instead.

File: docs/rst/introduction/metapackages.rst (hunk 4 above)
Comment (juanlofer-eprosima) [reply docs/rst/introduction/metapackages.rst:6]: But I don't think it's necessary to add such clarification, as whoever is not sure about the concept should probably understand after skimming the page and seeing all the `apt install` commands in it.

File: docs/rst/tutorials/core/ros2_tutorials.rst (hunk 7 above)
Comment (juanlofer-eprosima) [reply docs/rst/tutorials/core/ros2_tutorials.rst:6]: Slightly modified.

File: docs/index.rst (hunk 8 above)
Comment (juanlofer-eprosima) [reply docs/index.rst:51]: Changed all of them.

File: docs/index.rst [hunk 13]
```diff
@@ -11,27 +11,54 @@ Vulcanexus Documentation
   :target: http://www.vulcanexus.com/
 
 
-*Vulcanexus* is a *ROS 2* (Robot Operating System) all-in-one tool set.
```
Comment (rsanchez15) [inline docs/index.rst:14]: This cannot be changed.

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

**Review by rsanchez15 — APPROVED** [review]
LGTM

**Pull request merged**